### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#496] [Esami & Esoneri M-Z]
### Traccia

<img width="939" alt="image" src="https://github.com/user-attachments/assets/b6674893-fc9a-4218-ab8b-e24415acf1ea">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/40972020-898d-44e6-8c33-be62db3f6203)
